eng/build-utils.ps1

```diff
@@ -159,7 +159,20 @@ function Exec-Script([string]$script, [string]$scriptArgs = "") {
 
 # Ensure the proper .NET Core SDK is available. Returns the location to the dotnet.exe.
 function Ensure-DotnetSdk() {
-  return Join-Path (InitializeDotNetCli -install:$true) "dotnet.exe"
+  $dotnetInstallDir = (InitializeDotNetCli -install:$true)
+  $dotnetTestPath = Join-Path $dotnetInstallDir "dotnet.exe"
```

Comment:
Rather than having two uses of `Test-Path` why not just build the dotnet name based on the current OS? If it's windows then `dotnet.exe` else `dotnet`?

Reply:
Because it was really easy to use test-path, and I'm not sure how to test the current operating system in powershell :).